addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -49,8 +49,8 @@ patch(PosStore.prototype, {
         }
     },
     async actionAfterIdle() {
+        // FIXME: the open modals should be closed when the idle timer is triggered
         if (document.querySelector(".modal-open")) {
```

Comment:
This condition here is a problem. It should be "if not error modal open, we should proceed with the logic". However, this is preventing the logic to run because most of the time, there is no modal open.
Please fix this in this PR (in a different commit), referencing the bug was introduced in the dialog PR.